ouroboros-network/src/Ouroboros/Network/PeerSelection/KnownPeers.hs

```diff
@@ -242,37 +268,17 @@ setCurrentTime :: Ord peeraddr
                => Time
                -> KnownPeers peeraddr
                -> KnownPeers peeraddr
-setCurrentTime now knownPeers@KnownPeers { nextGossipTimes, nextConnectTimes }
```

Comment:
`setCurrentTime`, `minGossipTime` and `setGossipTime` are moved to `EstablishedPeers`, isn't it?

Reply:
Yes!